api/python/cellxgene_census/tests/experimental/pp/test_stats.py

```diff
@@ -0,0 +1,144 @@
+import math
+
+import pandas as pd
+import pytest
+import tiledbsoma as soma
+
+import cellxgene_census
+from cellxgene_census.experimental import pp
+
+
+@pytest.fixture
+def small_mem_context() -> soma.SOMATileDBContext:
```

Comment:
should move into conftest to keep it DRY (this is a cut & paste of the code in test_hvg.py)